gsonrudderadapter/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "gsonrudderadapter",
+  "version": "1.0.0",
```

Comment:
version seems wrong 1.0.1 (2023-07-13)

Reply:
I was thinking to test the complete flow with these values, then update them.